packages/node/test/helpers.ts

```diff
@@ -54,9 +63,15 @@ export class TestProcess {
   // a users requests an interrupt via CTRL-C or equivalent.
   interrupt() {
     if (this.isWin32) {
-      //TODO:
+      // @ts-ignore
+      this.process.kill("SIGINT");
+      // @ts-ignore
+      this.process.stdout.end();
+      // @ts-ignore
+      this.process.stderr.end();
```

Comment:
This is a f

Reply:
☝️  me trying to type a comment on my phone while lying in bed about 15 seconds after waking up and then falling back asleep 😂